infrastructure/test/docker-compose.e2e.yml

```diff
@@ -0,0 +1,16 @@
+version: "3"
```

Comment:
Not sure if I'm doing anything wrong, but I'm getting type orm errors when trying to run the code. I'm using node16, and yarn installed before running the tests. Not sure why its not working :(